pkg/types/types.go

```diff
@@ -474,15 +474,40 @@ type FluentdOperator struct {
 }
 
 type Filebeat struct {
+	Enabled
```

Comment:
Can we switch these all to Disabled?

i.e. if a Version is specified, then it is enabled, to disable it either remove the config, or specify:    
`-e filebeat.disabled=true`